_posts/2020-09-29-terraform-0-13-and-google-cloud.markdown

```diff
@@ -19,17 +19,17 @@ published: true
 
 ## **Terraform**
 
-Terraform, the de facto configuration tool for managing infrastructure nowadays has come a long way since its early release, roughly 5 years ago. After many iterations, mostly driven by the vocal supporters, one of terarrorm’s biggest strengths, listening to its community has proven judgmental to its success.
+Terraform, the de facto configuration tool for managing infrastructure nowadays has come a long way since its early release, roughly 5 years ago. After many iterations, mostly driven by vocal supporters, one of Terraform’s biggest strengths, listening to its community has proven crucial to its success.
 
-This blog post, is going to be about how I practically apply [Terraform's 0.13] features into one of the major IaaS providers [Google Cloud].
+This blog post is going to be about how I practically apply [Terraform's 0.13] features into one of the major IaaS providers [Google Cloud].
 
 ## **The Scenario**
 
 Having worked with both AWS and GCP, I have found that as usual, there are pros and cons to both cloud providers.
 
-One slightly simpler thing (perhaps better) than the other cloud providers is the coupling between G Suite and GCP. G Suite is acting as a central identity store that we can use in GCP’s Cloud Identity and Access Management (IAM). The coupling between the users in a company and its IaaS provider simplifies the management overhead. As an example, one can use G Suite emails or email groups to assign permissions to resources within GCP. This leads to interesting combinations that clearly define the responsibilities, and we benefit from heightened security because of said segregation of roles and duties.
+One slightly simpler thing (perhaps better) than the other cloud providers is the coupling between G Suite and GCP. G Suite is acts as a central identity store that we can use in GCP’s Cloud Identity and Access Management (IAM). The coupling between the users in a company and its IaaS provider simplifies the management overhead. As an example, one can use G Suite emails or email groups to assign permissions to resources within GCP. This leads to interesting combinations that clearly define responsibilities, and we benefit from heightened security because of said segregation of roles and duties.
```

Comment:
"is" trqbva da se mahne
```suggestion
One slightly simpler thing (perhaps better) than the other cloud providers is the coupling between G Suite and GCP. G Suite acts as a central identity store that we can use in GCP’s Cloud Identity and Access Management (IAM). The coupling between the users in a company and its IaaS provider simplifies the management overhead. As an example, one can use G Suite emails or email groups to assign permissions to resources within GCP. This leads to interesting combinations that clearly define responsibilities, and we benefit from heightened security because of said segregation of roles and duties.
```